src/text/download.md

```diff
@@ -107,9 +107,17 @@ issue the following to find all available packages with LeechCraft
 modules:
 
 ``` {type="bash"}
-yum search leechcraft
+dnf search leechcraft
 ```
 
+\
+\
+Popular RPM Sphere repo can be used to get LeechCraft packages too.\
+\
+If you feel like the packages and/or releases are too old, don't hesitate [filling an
+issue](http://dev.leechcraft.org/projects/leechcraft/issues/new) against
+DA for a version bump.
```

Comment:
`against` is rather about a subject than a person. A better one would be `and assigning it to DA`.